kie-drools-wb/kie-drools-wb-home-page-community/src/main/java/org/kie/workbench/drools/client/home/HomeProducer.java

```diff
@@ -23,7 +23,7 @@
 import org.kie.workbench.common.screens.home.model.HomeModel;
 import org.kie.workbench.common.screens.home.model.ModelUtils;
 import org.kie.workbench.common.screens.home.model.SectionEntry;
-import org.kie.workbench.common.screens.library.client.monitor.LibraryMonitor;
+import org.kie.workbench.common.workbench.client.library.LibraryMonitor;
```

Comment:
@psiroky Indeed it is redundant... Best to leave the change to another PR, since the package already existed and is used by all webapps. I opened a [JIRA issue](https://issues.jboss.org/browse/GUVNOR-2871) for that.